bcipy/simulator/sim_factory.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+from bcipy.config import DEFAULT_PARAMETER_FILENAME
+from bcipy.simulator.sim_copy_phrase import SimulatorCopyPhraseReplay
+from bcipy.simulator.simulator_base import Simulator
+
+
+class SimulationFactory:
+
+    @staticmethod
+    def create(
+            sim_task="",
+            parameter_path="",
+            smodel_files=None,
+            lmodel_files=None,
+            data_folders=None,
+            out_dir="",
+            **kwargs) -> Simulator:
+        if sim_task == 'RSVP_COPY_PHRASE':
```

Comment:
See the task_registry module for supported tasks. There is an Enum in there that would be useful here for dispatching.